rasa/nlu/training_data/training_data.py

```diff
@@ -125,22 +182,78 @@ def sort_regex_features(self) -> None:
             self.regex_features, key=lambda e: "{}+{}".format(e["name"], e["pattern"])
         )
 
-    def as_json(self, **kwargs: Any) -> Text:
+    def fill_response_phrases(self):
+        """Set response phrase for all examples by looking up NLG stories"""
+        for example in self.training_examples:
+            response_key = example.get(MESSAGE_RESPONSE_KEY_ATTRIBUTE)
+            if response_key:
+                # look for corresponding bot utterance
+                story_lookup_intent = example.get_combined_intent_response_key()
+                assistant_utterances = self.nlg_stories.get(story_lookup_intent, [])
+                if assistant_utterances:
+                    # selecting only first assistant utterance for now
+                    example.set(MESSAGE_RESPONSE_ATTRIBUTE, assistant_utterances[0])
+                else:
+                    raise ValueError(
+                        "No response phrases found for {}. Check training data "
+                        "files for a possible wrong intent name in NLU/NLG file".format(
+                            story_lookup_intent
+                        )
+                    )
+
+    def nlu_as_json(self, **kwargs: Any) -> Text:
         """Represent this set of training examples as json."""
         from rasa.nlu.training_data.formats import (  # pytype: disable=pyi-error
             RasaWriter,
         )
 
         return RasaWriter().dumps(self, **kwargs)
 
-    def as_markdown(self) -> Text:
```

Comment:
for compatibility reasons I'd keep this as a deprecated function return `nlu_as_markdown`